src/Raven.Server/Rachis/Leader.cs

```diff
@@ -57,6 +57,8 @@ public class Leader : IDisposable
         private readonly ConcurrentDictionary<string, FollowerAmbassador> _nonVoters =
             new ConcurrentDictionary<string, FollowerAmbassador>(StringComparer.OrdinalIgnoreCase);
 
+        public Dictionary<string, FollowerAmbassador> CurrentPeers => new  Dictionary<string, FollowerAmbassador>(_voters.Concat(_nonVoters).Concat(_promotables));
```

Comment:
What updates this? What happens when we add a new node? Remove one?

Reply:
`RefreshAmbassadors` will update this and handle add/remove.

Reply:
This is only called from the tests, right?

Reply:
Yes, but maybe we should add a comment 'DEBUG ONLY', just in case